websocket-compiler-plugin/src/main/java/io/ballerina/stdlib/websocket/plugin/PluginConstants.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.stdlib.websocket.plugin;
+
+/**
+ * WebSocket compiler plugin constants.
+ */
+public class PluginConstants {
+
+    public static final String COLON = ":";
+    public static final String SERVICE = "Service";
+    public static final String PIPE = "|";
+    public static final String UPGRADE_ERROR = "UpgradeError";
+    static final String CODE = "WS_110";
+    static final String ON_ERROR = "onError";
+    static final String ON_OPEN = "onOpen";
+    static final String ON_CLOSE = "onClose";
+    static final String ON_IDLE_TIMEOUT = "onIdleTimeout";
+    static final String ON_TEXT_MESSAGE = "onTextMessage";
+    static final String ON_BINARY_MESSAGE = "onBinaryMessage";
+    static final String REMOTE_KEY_WORD = "remote";
+    static final String RESOURCE_KEY_WORD = "resource";
+    static final String HTTP_REQUEST = "http:Request";
+
+    /**
+     * Compilation Errors of WebSocket module.
+     */
+    public enum CompilationErrors {
+        INVALID_INPUT_PARAM_FOR_ON_CLOSE("Invalid parameters `{0}` provided for onClose remote function",
+                "WS_202"),
+        INVALID_INPUT_FOR_ONCLOSE_WITH_ONE_PARAMS("Invalid parameters `{0}` provided for onClose remote "
+                + "function. `string` is the mandatory parameter", "WS_203"),
```

Comment:
We haven't used WS in any other places right ? I think in all the places we have used WebSocket ?

Reply:
Updated the error codes to `WEBSOCKET_xxx` with https://github.com/ballerina-platform/module-ballerina-websocket/pull/107/commits/6c6eec5cbd8ec2c4870a1049efca0f13339568b1